sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -438,6 +438,19 @@ export class ServiceBusClient {
       options3
     );
 
+    const operationTimeout =
+      this._clientOptions?.retryOptions?.timeoutInMs ?? Constants.defaultOperationTimeoutInMs;
+    // The number of milliseconds to use as the basis for calculating a random jitter amount
+    // opening receiver links. This is intended to ensure that multiple
+    // session operations don't timeout at the same exact moment.
+    const jitterBaseInMs = Math.min(operationTimeout * 0.01, 100);
```

Comment:
The jitter may not be as useful in JS since we don't have multiple threads.

Reply:
Well there is still the concept of concurrency, no? It may all be occurring on one thread but I think spreading out the timeouts would still help alleviate context switching.